includes/Admin/Admin_AJAX.php

```diff
@@ -278,6 +223,70 @@ public function run_checks() {
 		);
 	}
 
+
+	/**
+	 * Get the AJAX_Runner runner.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return AJAX_Runner
+	 */
+	protected function get_runner() {
+		$runner = Plugin_Request_Utility::get_runner();
+
+		if ( is_null( $runner ) ) {
+			Plugin_Request_Utility::initialize_runner();
+			$runner = Plugin_Request_Utility::get_runner();
```

Comment:
The `initialize_runner` method handles other logic alongside creating the correct runner (and could handle more in the future). As we know we're in the AJAX context I think we should simply instantiate an `AJAX_Runner` as before rather than use the `Plugin_Request_Utility` methods here.

Reply:
Then what is the point of ` Plugin_Request_Utility::get_runner();` then. Why not just create a `AJAX_Runner` here?

Reply:
+1 to what @jjgrainger said, this is changing the logic from what it was before. We should create an `AJAX_Runner` here, not call `Plugin_Request_Utility::initialize_runner()` and `Plugin_Request_Utility::get_runner()`, as that also runs other logic.